README.md

```diff
@@ -29,6 +29,18 @@ $ puma
 Without arguments, puma will look for a rackup (.ru) file in
 working directory called `config.ru`.
 
+## SSL Connection Support
+
+Puma will install/compile with support for ssl sockets, assuming OpenSSL
+development files are installed on the system.
+
+If the system does not have OpenSSL development files installed, Puma will
+install/compile, but it will not allow ssl connections.
+
+If the system has OpenSSL development files installed, but you don't want Puma
```

Comment:
I feel like this is a little unclear. If you don't bind Puma to SSL, why should you need to use `DISABLE_SSL`?

As written, this makes it sound like anyone using Puma w/o SSL _must_ set `DISABLE_SSL`.

Reply:
> If you don't bind Puma to SSL, why should you need to use DISABLE_SSL?

You don't have to, but 'If the system has OpenSSL development files', it will compile the SSL functions into `puma_http11.so/jar` and will load the OpenSSL libraries/dlls when using MRI.  It also loads Ruby OpenSSL on MRI. 
 I believe using `DISABLE_SSL` stops all of that.

> this makes it sound like anyone using Puma w/o SSL must set DISABLE_SSL.

Didn't mean to imply that.  Maybe a rephrasing is in order.  I meant to make it clear that it's optional.  But, I didn't say anything about benefits...

BTW, thanks for reviewing.

Reply:
Yeah I just ended up removing it. I think it's too confusing. Good feature to have but not one that needs explaining right in README.md. 

Thanks so much for your work on the test suite over the last 2 weeks Greg, it's so much better now.